docs/usecases/datastorage_for_institutions.rst

```diff
@@ -0,0 +1,314 @@
+progress, despite the elaborate and expensive cluster setup.
+
+The DataLad approach
+^^^^^^^^^^^^^^^^^^^^
+
+The compute cluster is refurbished to a state-of-the-art data management
+system. Unlike traditional solutions, both because of the size of the large
+amounts of data, and for more efficient use of compute power for
+calculations instead of data storage, the cluster gets a distributed data
+store: Data lives as DataLad datasets on a different machine than the one
+the scientific analyses are computed on.
+For access to the annexed data in datasets, the data store is configured as a
+Git-annex `RIA-remote <https://libraries.io/pypi/ria-remote>`_.
+In case of filesystem inode limitations on the machine
+serving as the data store (e.g., HPC storage systems), full datasets can be
+(compressed) 7-zip archives, without loosing the ability to query available files.
+Regardless of file number and size, such datasets thus use only few inodes.
+Using DataLad's run-procedures, an institute-wide
+configuration is distributed among users. Applying the procedure is done in a
+single :command:`datalad run-procedure` command, and users subsequently
+face minimal technical overhead to interact with the data store.
+
+The infrastructural changes are accompanied by changes in the mindset and workflows
+of the researchers that perform analyses on the cluster.
+By using the data store, the institutes work routines are adjusted around
+DataLad datasets: Analyses are set-up inside of DataLad datasets, and for every
+analysis, an associated ``project`` is created under the namespace of the
+institute on the institute's :term:`Gitlab` instance automatically. This has
+the advantage of vastly simplified version control, even for trainees, and
+simplified access to projects for collaborators and supervisors. Data
+from the data store is installed as subdatasets. This comes with several
+benefits: Analyses are automatically linked to data, no unused file
+copies waste disk space on the compute cluster as data can be retrieved
+on-demand, and files that are easily re-obtained or recomputed can safely be
+dropped to save even more disk-space. More over, upon creation of an analysis
+project, the associated Gitlab project it is automatically configured as a remote
+with a publication dependency on the data store, thus enabling vastly simplified
+data publication routines and back-up of pristine results.
+
+
+Step-by-step
+^^^^^^^^^^^^
+
+.. note::
+
+   This use case describes the data storage implementation as done in INM-7,
+   research centre Juelich, Germany.
+
+To create a data store, parts of the old compute cluster and parts of the
+super computer at the Juelich supercomputing centre (JSC) are used to store
+large amounts of data. Thus, multiple different, independent machines take care of
+warehousing the data. While this is unconventional, it is convenient: The
+data does not strain the compute cluster, and with DataLad, it is irrelevant
```

Comment:
I wouldn't make a judgement here of where the data should or shouldn't be. It's a complex matrix of space, disk I/O, fragmentation, network I/O, etc. What DataLad /does/ do is give you options to store/move data where the admin thinks is best, without having the need for an insane, superhuman ability to track every one of those ~1 billion files.